### PR TITLE
Use env vars in worker.config.json

### PR DESCRIPTION
**Pull request by pragnagopa**
Corresponding Host change: Azure/azure-functions-host#6231

This PR allows using environment variables in worker.config.json and also removes java specific code from host - giving more control to java worker on customizing input args for java worker.

### CODE REVIEW

**Comment by pragnagopa** [comment]
@amamounelsayed  @TsuyoshiUshio - can you please with releasing a test java worker with this change to allow for tests to run on host pr?

**Comment by amamounelsayed** [comment]
@pragnagopa Thank you so much for your update, for now the only way to create test release is to disable the tests as it depends on the previous version of host. Do you agree? if yes I will go ahead and do it on this branch.

**Comment by pragnagopa** [comment]
> disable the tests as it depends on the previous version of host

yes, please do. Then I will update the host PR pointing to test jar - this will ensure e2e tests are ok, then we need to create prod version of nuget.

File: setup-tests-pipeline.ps1 [hunk 1]
```diff
@@ -17,13 +17,13 @@ if(!$skipCliDownload)
...
   Invoke-RestMethod -Uri 'https://functionsclibuilds.blob.core.windows.net/builds/3/latest/version.txt' -OutFile version.txt
-  Write-Host "Using Functions Core Tools version: $(Get-Content -Raw version.txt)"
+  Write-Host "Using Functions Core Tools version: pgopa-CLI"
   $version = "$(Get-Content -Raw version.txt)"
   Remove-Item version.txt
 
   if ($version -and $version.trim())
   {
-    $env:CORE_TOOLS_URL = "https://functionsclibuilds.blob.core.windows.net/builds/3/latest/Azure.Functions.Cli.win-x86.zip"
+    $env:CORE_TOOLS_URL = "https://pgopafunctestv2storage.blob.core.windows.net/cli/pgopa-cli.zip"
```
Comment (TsuyoshiUshio) [inline setup-tests-pipeline.ps1:26]: Where can I find the storage account? Is this published by the Azure Functions Core Tools CI?

Comment (amamounelsayed) [reply setup-tests-pipeline.ps1:26]: We just pushed the latest host from Pragna branch as we did not release the latest host yet.
We will go back and update these values once we will release the core tools in few weeks.

Comment (pragnagopa) [reply setup-tests-pipeline.ps1:26]: Yes, core tools ci builds are uploaded to 'functionsclibuilds'. I will revert this as soon as the latest host changes move to release

**Pull request merged**